docs/_includes/footer.html

```diff
@@ -1,24 +1,9 @@
-<footer class="site-footer">
-    <div class="wrapper">
-        <div class="footer-col-wrapper">
-            <div class="footer-col footer-col-1">
-                <ul class="contact-list">
-                    <li>{{ site.title }}</li>
-                    <li><a href="http://groups.google.com/group/sinonjs">http://groups.google.com/group/sinonjs</a></li>
```

Comment:
Yeah, I forget to add the google group 😅 . Will proceed it now, and add it to the footer.